helix-query.yaml

```diff
@@ -41,3 +41,30 @@ indices:
       cq-tags:
         select: head > meta[name="cq-tags"]
         value: attribute(el, "content")
+  partners:
+    include:
+      - /partner/**
```

Comment:
@mayur-satav Can you confirm if this is partner or partners?

Reply:
i think as per the screenshot Munnesh shared it is the partners

Reply:
@muneeskumar-rajendran Can you confirm the same? 

@mayur-satav Can you update it to partners if @muneeskumar-rajendran confirms that?